install_config/master_node_configuration.adoc

```diff
@@ -856,10 +856,13 @@ To enable the advanced audit feature, you must provide either `policyFile` or
 .Sample Audit Policy Configuration
 [source,yaml]
 ----
-apiVersion: audit.k8s.io/v1alpha1
+apiVersion: audit.k8s.io/v1beta1
```

Comment:
@soltysh - looking.

Reply:
@soltysh - Hey Maciej, looks like there was some confusion around whether the change in #6662 applied to 3.7 and/or 3.9 branches. See the note by @gaurav-nelson here: https://github.com/openshift/openshift-docs/pull/6496#issuecomment-363226706. which didn't get a clarification. 

This was around the time when we were switching workflows as well so that may have contributed to it as well.

So to confirm, https://github.com/openshift/openshift-docs/pull/6662 should apply to 3.7 and NOT to 3.9? And that the only change from #6662 that you don't see in the 3.7 branch is the change from v1alpha1 to v1beta1?

Reply:
3.7 is where audit is in alpha and tech preview, so #6662 should be re-applied there and only there.

Reply:
@vikram-redhat ^

Reply:
Thanks @soltysh - will fix it today.

Reply:
@soltysh - fixed via: https://github.com/openshift/openshift-docs/commit/74e530c3261baa707cbde9c1a488f97f79e247ef. Should be live in the next 20 minutes.

Reply:
Thanks!